### PR TITLE
fix: overlapping footer layout of `browse` and `create` buttons

### LINKED ISSUE #1424: 🐛 [Bug report]:  UI Issues in various small and medium sized devices
### Descrcibe the bug?

The browse and create button on the footer overlaps other content when viewed on different small to medium sized devices.
This issue does not persist with every such device but with some of the devices.
I am attaching an image for the same with various devices that I have found.

Sol: The best solution is to move the email us and Netlify logo in a container, make it a flexbox, and move the browse and create button just below that container.
This way it will work fine on all the devices and will not overlap other contents on the footer.

Untitled

The image added contains a screenshot from 3 devices on which I have found the bug. There might be others too.

### Steps to reproduce the bug?

If you are using the website on a desktop, then switch to responsive view and try switching between the available devices.

If you are using a small to medium-sized device then just scroll down to the footer section and the buttons that are overlapping will be just there.

This issue is only with some of the small to medium-sized devices so you might not see this issue on your device.

### Expected behavior

The browse and create button should not be overlapped with other footer content.

### Desktop (Please provide your system information)

Laptop: Acer Nitro 5
OS: Windows 11
Browser: Chrome version 120.0.6099.225 (Official Build) (64-bit)

### Mobile (Please provide your device information)

Mobile: Samsung f54
OS: Android 14
UI: One UI 6.0

### Screenshot / Screenshare

Untitled

### Relevant log output

_No response_

### Record

- [X] I have checked the existing [issues](https://github.com/reactplay/react-play/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/reactplay/react-play/blob/main/CONTRIBUTING.md)
- [X] I want to work on this issue

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by clevercoderjoy**
Layout for create and browse button fix for small and medium-sized devices

# Description

Fixed the overlapping create and browse buttons in the footer section for all small and medium-sized devices.

Fixes #1424

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- This has been tested with the Chrome dev tools responsive view.
- Switched between different devices and checked that the UI was not breaking.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

image
image

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
There is a conflict error in this branch. Could you take steps to resolve it?

**Comment by clevercoderjoy** [comment]
> There is a conflict error in this branch. Could you take steps to resolve it?

File: package.json [hunk 1]
```diff
@@ -36,8 +36,10 @@
     "leaflet": "^1.9.4",
     "lodash": "^4.17.21",
     "lodash.memoize": "^4.1.2",
-    "mathjs": "^11.11.2",
-    "p5": "^1.7.0",
+    "mathjs": "^11.12.0",
+    "open": "^10.0.3",
```
Comment (priyankarpal) [inline package.json:40]: remove the package 
```suggestion

```

File: package.json [hunk 2]
```diff
@@ -36,8 +36,10 @@
...
+    "p5": "^1.9.0",
+    "plays": "^0.7.0",
```
Comment (priyankarpal) [inline package.json:42]: remove this too

```suggestion

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (clevercoderjoy) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (clevercoderjoy) [reply package.json]: this too done

**Review by priyankarpal — CHANGES_REQUESTED** [review]
there are little bit issue
image


Screenshot_2024-02-11_10-16-47

**Comment by priyankarpal** [comment]
now it's looking good ✨

**Review by priyankarpal — DISMISSED** [review]

**Comment by priyankarpal** [comment]
@all-contributors add @clevercoderjoy for code

**Review by priyankarpal — CHANGES_REQUESTED** [review]
everything is messed up. Could you please create a separate PR?

**Pull request closed without merging**